modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/layout_customization_settings.js

```diff
@@ -181,14 +179,19 @@ AUI.add(
 					var customizationsHandle = instance._customizationsHandle;
 
 					if (!customizationsHandle) {
-						customizationsHandle = BODY.delegate(
-							EVENT_CLICK,
-							instance._onChangeCustomization,
-							'.layout-customizable-checkbox',
-							instance
-						);
+						document.body.addEventListener(EVENT_CLICK, (event) => {
```

Comment:
Same with this delegate. Let's just use an eslint-ignore for the specific no-body rule